### DISCUSSION TITLE
sidebar/sanfene/fenbushi

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#280] [Announcements]
# sidebar/sanfene/fenbushi

下载次数超 1 万次，8800 字 25 张手绘图，详解 12 道分布式面试高频题（让天下没有难背的八股），面渣背会这些八股文，这次吊打面试官，我觉得稳了（手动 dog）。

https://javabetter.cn/sidebar/sanfene/fenbushi.html

**Comment by Kddk-K** [comment]
9.12一刷
